docs/UserManual.md

```diff
@@ -0,0 +1,28 @@
+# AAFC-DINA User Manual
+
+## Groups
+
+In DINA, groups are used to define a group of user. Mainly a natural history collection or a lab.
+A user can belong to more than 1 group.
+
+Groups are also used to defined who own an object. For example, an object in the object store always belong to a group.
```

Comment:
Groups are also used to defined who owns an object. For example, an object in the object store always belongs to a group.